helm/eirini/templates/namespace.yaml

```diff
@@ -4,3 +4,18 @@ kind: Namespace
 metadata:
   name: {{ .Values.opi.namespace }}
 
+{{- if and (eq (printf "%s" .Values.kube.auth) "rbac") (.Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1") }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: eirini-nonprivileged
+  namespace: {{ .Values.opi.namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "fissile.SanitizeName" (printf "%s-cluster-role-nonprivileged" .Release.Namespace) }}
```

Comment:
I'm not exactly sure on this one ^. I'm trying to reuse what already exists (nonprivileged role created by `cf` chart) however it might make sense to create a more restricted role just for this namespace in the future.

Reply:
This doesn't work anymore after because eirini [applies](https://github.com/cloudfoundry-incubator/eirini/commit/88aae699e8f0bdd888b3fc128d685664a09710ee) `seccomp` now and `psp` coming from scf don't have seccomp yet.